File: system-test/sec-tests/python/sec_17_csrf.py
# SEC-17: CSRF Protection Missing
# Test: TC-CR-024
from flask import Flask, request

app = Flask(__name__)

@app.route('/transfer', methods=['POST'])
def transfer():
    amount = request.form['amount']
    to_account = request.form['to']
    # No CSRF token validation
    make_transfer(amount, to_account)
    return 'OK'

File: system-test/sec-tests/javascript/sec_17_csrf.js
// SEC-17: CSRF Protection Missing
// Test: TC-CR-024
const express = require('express');
const app = express();

app.post('/transfer', (req, res) => {
  const { amount, to } = req.body;
  // No CSRF token validation
  makeTransfer(amount, to);
  res.send('OK');
});
